main/main_timer_sync.cpp

```diff
@@ -40,6 +42,220 @@ void MainFrameTime::clamp_idle(float min_idle_step, float max_idle_step) {
 
 /////////////////////////////////
 
+void MainTimerSync::DeltaSmoother::update_refresh_rate_estimator(int p_delta) {
+	// this should have been taken care of earlier in code, preventing 0 or small values of delta
+	CRASH_COND(!p_delta);
```

Comment:
I recommend adding a message in case someone needs to report a bug:

```suggestion
	CRASH_COND_MSG(!p_delta, "Delta smoothing: Delta value is too small (please report).");
```

Reply:
Have changed this to a comment only. For things like this I would typically use a developer only debug assert, and Godot doesn't have this feature, as the editor is built with release_debug (although I work around this in the renderer if I remember right).